canonical/rebac-admin
# WD-6744 - Application layout

## Done

- Add example application layout

## QA

- Open the demo (or run `yarn start` locally).
- Check that the application layout appears.

## Details

https://warthogs.atlassian.net/browse/WD-6744

## Screenshots

<img width="1508" alt="Screenshot 2023-10-25 at 4 09 51 pm" src="https://github.com/canonical/rebac-admin/assets/361637/aaf9be77-8478-4ad4-a611-bd14956c4a08">
